src/components/UploadEditor.vue

```diff
@@ -45,14 +45,16 @@
 					</template>
 					<div :key="'addMore'"
 						class="add-more">
-						<button :aria-label="addMoreAriaLabel"
-							class="add-more__button primary"
+						<Button :aria-label="addMoreAriaLabel"
```

Comment:
I don't think we want to change this one @nikola-gladovic. The original is much bigger than 44px right? Could you please add screenshots of how the button looked before the swap and after on all of your open button pull requests?

Reply:
Ok, I will do that. @marcoambrosini  :)

Reply:
Thanks!